allocation_manager/service/unit.go

```diff
@@ -46,18 +49,56 @@ metricLicenseReleaseReason = promauto.NewCounterVec(prometheus.CounterOpts{
 	},
 )
 */
+
+unitCounter = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "unit_operations_total",
```

Comment:
we should probably prefix all our metric names with allocation_manager_ like how service.go does, i.e. allocation_manager_request_duration_seconds